### PR TITLE
feat(ui): Small react changes

### PR DESCRIPTION
**Pull request by billyvg**
- Fix `SettingsPageHeader` title proptype
- Allow `Form` to hide footer
- Add some emotion theme defs
- Add "enroll" mode to `U2fSign`
- Allow CSS style obj to be passed to `U2fInterface`
- Increase default settings indicator message duration
- Add error boundary to `LazyLoad`
- Fix class method binding in `AsyncComponent`

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/utils/theme.js [hunk 1]
```diff
@@ -93,6 +94,9 @@ const theme = {
   },
 };
 
+// Aliases
 theme.textColor = theme.gray5;
+theme.disabled = theme.red;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils/theme.js]: These (disabled/enabled) are bad names, looking for suggestions

Comment (lynnagara) [reply src/sentry/static/sentry/app/utils/theme.js]: success / fail? That's what's comes to mind for red/green, disabled to me is more like grey or something

File: src/sentry/static/sentry/app/components/asyncComponent.jsx [hunk 2]
```diff
@@ -51,12 +51,12 @@ class AsyncComponent extends React.Component {
     return state;
   }
 
-  remountComponent() {
+  remountComponent = () => {
     this.setState(this.getDefaultState(), this.fetchData);
-  }
+  };
 
   // TODO(dcramer): we'd like to support multiple initial api requests
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/asyncComponent.jsx]: can we drop this todo now?

File: src/sentry/static/sentry/app/constants.jsx [hunk 3]
```diff
@@ -30,4 +30,4 @@ export const DEFAULT_API_SCOPES = [
   'member:read',
 ];
 
-export const DEFAULT_TOAST_DURATION = 2000;
+export const DEFAULT_TOAST_DURATION = 6000;
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/constants.jsx]: 👍

**Review by MaxBittker — APPROVED** [review]
very nice. 🎁

**Pull request merged**